File: chap3/R/code5.r
x = layer_input(784)
z1 = x %>% layer_dense(units = 256, activation="relu")
dropout = z1 %>% layer_dropout(rate = 0.25)
z2 = dropout %>% layer_dense(units = 128, activation="relu")
y = z2 %>% layer_dense(units = 10, activation="softmax")
model = keras_model(x, y)

File: chap3/julia/code5.jl
using Flux

z1 = Dense(784, 256, relu)
dropout = Dropout(0.25)
z2 = Dense(256, 128, relu)
y = Dense(128, 10)
model = Chain(
  dropout,
  z1, 
  z2, 
  y, 
  softmax
)
